ocp_resources/datavolume.py

```diff
@@ -87,8 +87,39 @@ def __init__(
         yaml_file=None,
         delete_timeout=TIMEOUT_4MINUTES,
         api_name="pvc",
+        delete_after_completion=None,
         **kwargs,
     ):
+        """
+        DataVolume object
+
+        Args:
+            eviction (bool, default False): If True, set evictionStrategy to LiveMigrate explicitly.
```

Comment:
please order the docstring by args orders

Reply:
this is be mistake, fixed